code/web/src/modules/user/api/actions.js

```diff
@@ -27,7 +27,37 @@ export function setUser(token, user) {
 
 // Login a user using credentials
 export function login(userCredentials, isLoading = true) {
-    return (dispatch) => {
+  return dispatch => {
```

Comment:
Not sure why this is showing up like this, I believe this is what we already have in main

Reply:
This is the same action that I was prompted to solve conflicts. It should be here anyways